main.tf

```diff
@@ -250,6 +250,10 @@ locals {
 }
 
 resource "null_resource" "create_yugabyte_universe" {
+  # Define the trigger condition to run the resource block
```

Comment:
Just so I understand, will this cause all the nodes to "reinstall" yugabyte db on them? Or will only the new node run this?

Reply:
all the nodes, but most of the other nodes it should be a noop